pyfar/dsp/__init__.py

```diff
@@ -18,8 +18,8 @@
 from .dsp_interpolation import (
     smooth_fractional_octave,
     fractional_delay_sinc,
-    InterpolateSpectrum,
-    resample
+    resample,
+    InterpolateSpectrum
 )
```

Comment:
Please refrain from cosmetic changes like this. This has potential to mess up git diffs and may lead to merge conflicts.